db/migrate/20150416085059_rename_city_and_address3_column_name.rb

```diff
@@ -0,0 +1,16 @@
+class RenameCityAndAddress3ColumnName < ActiveRecord::Migration
```

Comment:
why mixing change method and up and down methods this can be done in one method change